### PR TITLE
feat: added Box and ToggleBox components

### PR DESCRIPTION
**Pull request by braianj**
Added ToogleBox component that use Box Component
Captura de Pantalla 2022-04-18 a la(s) 13 06 51
Captura de Pantalla 2022-04-18 a la(s) 13 28 26

### CODE REVIEW

**Comment by 2fd** [comment]
the new component should have the same alignment that currently have in the market 

| market | current |
|--------|---------|
| market | toggle-box|

**Comment by braianj** [comment]
Thanks @2fd. 
The new components were aligned wi
Captura de Pantalla 2022-04-18 a la(s) 15 22 19
Captura de Pantalla 2022-04-18 a la(s) 15 22 57
th the filter component by type found in the marketplace

File: src/components/ToggleBox/ToggleBox.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import classNames from 'classnames'
+import { Box } from '../Box/Box'
+import './ToggleBox.css'
+
+export type ToggleBoxItem = {
+  title: string
+  description: string
+  active?: boolean
+  disabled?: boolean
+  onClick?: (item: ToggleBoxItem, index: number) => unknown
```
Comment (2fd) [inline src/components/ToggleBox/ToggleBox.tsx:11]: please move this into ToggleBoxProps

File: src/components/ToggleBox/ToggleBox.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+export type ToggleBoxProps = {
+  header: string
+  className?: string
+  items: ToggleBoxItem[]
+}
```
Comment (2fd) [inline src/components/ToggleBox/ToggleBox.tsx:18]:
```suggestion
  onClick: (event: React.MouseEvent<any>, item: ToggleBoxItem) => void
}
```

**Review by 2fd — CHANGES_REQUESTED** [review]

File: src/components/Box/Box.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import classNames from 'classnames'
+import './Box.css'
+
+export type BoxProps = {
+  header?: string
+  className?: string
+  children: React.ReactNode
+  childrenClassName?: string
```
Comment (cazala) [inline src/components/Box/Box.tsx:9]: is it necessary to have this children class name? what's the use case? couldn't it be themed using the other `className`? Like, if I wanted to customize this particular children's rules I could pass `className="my-custom-class"` and then `.my-custom-class .dcl.box-children { ... }`

File: src/components/ToggleBox/ToggleBox.tsx (hunk 2 above)
Comment (braianj) [reply src/components/ToggleBox/ToggleBox.tsx:21]: change applied

File: src/components/Box/Box.tsx (hunk 3 above)
Comment (braianj) [reply src/components/Box/Box.tsx]: Yes, you are right. Thanks for the clarification. It has been removed

**Review by 2fd — APPROVED** [review]

**Review by cazala — APPROVED** [review]

**Comment by cazala** [comment]
LGTM!

**Pull request merged**